include/af/defines.h

```diff
@@ -446,6 +446,13 @@ typedef enum {
 #endif
 ////////////////////////////////////////////////////////////////////////////////
 
+#if AF_API_VERSION >= 35
+typedef enum {
+    AF_MANUAL_THRESHOLD    = 0, ///< User has to define canny thresholds manually
+    AF_AUTO_OTSU_THRESHOLD = 1, ///< Determine canny algorithm thresholds using Otsu algorithm
+} af_canny_threshold;
```

Comment:
I don't think Canny edge detection uses any other type of thresholding. Do we really need an enum?